Nodejs/Product/Nodejs/NodejsConstants.cs

```diff
@@ -17,6 +17,7 @@ internal static class NodejsConstants
         public const string NodejsProjectExtension = ".njsproj";
         public const string CSharpProjectExtension = ".csproj";
         public const string VisualBasicProjectExtension = ".vbproj";
+        public const string AngularProjectExtension = ".esproj";
```

Comment:
Is `.esproj` the new JSPS project extension? If so, let's not specifically refer to it as "Angular".

Reply:
Agree. Any suggested name for it?

Reply:
I think either `JavaScriptProjectExtension` to match our common name for it (JSPS) or `ECMAScriptProjectExtension` to patch the `es` part is fine.

Reply:
`JavaScriptProjectExtension` it is. Done.